src/applications/disability-benefits/2346/Components/ConfirmAddress.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getVeteranAddressInfo } from '../actions';
+
+class ConfirmAddress extends Component {
+  componentDidMount() {
+    this.props.getVeteranAddressInfo();
+  }
+
+  render() {
+    const { addressLine1, addressLine2, city, state, zip, email } = this.props;
+
+    return (
+      <div>
+        <h1>Confirm your address</h1>
+        <div>
+          <label htmlFor="address1">Address Line 1</label>
+          <input id="address1" type="text" defaultValue={addressLine1} />
+          <label htmlFor="address2">Address Line 2</label>
+          <input type="text" defaultValue={addressLine2} />
+          <label htmlFor="city">City</label>
+          <input type="text" id="city" defaultValue={city} />
+          <label htmlFor="state">State</label>
+          <input type="text" id="state" defaultValue={state} />
+          <label htmlFor="zip">Zip</label>
+          <input type="text" id="zip" defaultValue={zip} />
+          <br />
+          <p>Confirm your email address</p>
+          <input type="text" defaultValue={email} />
+        </div>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  addressLine1: state.form2346Reducer?.formData?.veteranAddress?.street || '',
```

Comment:
Instead of doing `|| ''` at the end, you could just add default props for all of these. Or do like Brad suggested and ensure the initial values in the `form2346Reducer` are empty strings 🙂

Reply:
@mr0sari0  Please take care of this.

Reply:
@kelsonic @mojaray2k This is completed.